modules/pubmaticBidAdapter.js

```diff
@@ -217,6 +219,108 @@ function _checkParamDataType(key, value, datatype) {
   }
 }
 
+function _createNativeRequest(params) {
+  var nativeRequestObject = {
+    assets: []
+  };
+  for (var key in params) {
+    var assetObj = {};
+    if (!(nativeRequestObject.assets && nativeRequestObject.assets.length > 0 && nativeRequestObject.assets.hasOwnProperty('key'))) {
+      switch (key) {
+        case 'title':
+          assetObj = {
+            id: 1,
+            req: params[key].required,
+            title: {
+              len: params[key].length,
```

Comment:
is length a mandatory parameter here? if not then will setting it undefined cause issue?

Reply:
yes length is mandatory param, if it is not passed call we get no bids. There will be parsing error at adserver.

Reply:
okay, then we should add a check on mandatory parameter here in JS code?